README.md

```diff
@@ -19,4 +19,4 @@ The *"List of Prank Call Scripts"* project aims to create a platform where users
 
 - **Category Filter**: Users can filter prank scripts based on categories such as funny, scary, romantic, allowing for easy selection based on their mood or preferences.
 
-- **Share the Scripts**: Users can easily share their favorite prank call scripts to external platforms.
\ No newline at end of file
+- **Share the Scripts** : Users can easily share their favorite prank call scripts to external platforms. 
```

Comment:
Consider adding a newline at the end of the file. This is a common best practice to ensure that the file can be easily read by different tools. [medium]